## Fields
# Time that this direction vector is for
time timestamp
# ID of the measured RGV
int8 rgv_id
# Source of this direction vector
int8 measurement_source
# The direction vector
float64[3] direction
